### ISSUE TITLE
Alps parameters changing

### ISSUE THREAD

**Issue by iskakoff** [ALPSCore/ALPSCore#334]
After the save method in alps::params has been changed to

    void params::save(hdf5::archive& ar) const
        {
            possibly_parse();
            const std::string context=ar.get_context();
            const std::string dict_group=context+"/dictionary";
            const std::string def_group=context+"/definitions";
            const std::string state_group=context+"/object_state";
            ....

at least two codes I know about was broken. Is there any ongoing plans to change the hdf5 interface in future?

**Comment by galexv** [comment]
Yes, for sure, in the coming version the parameter storage format will change.
I am open to suggestions what could be a most desirable layout.
